### ISSUE TITLE
searching with conjunction characters searches for terms separately?

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud.solr#78]
I think I'm still confused how rcloud.solr deals with `.` and `_`. When I search for something like `show_vars` it looks like I get results for `show OR vars`.

**Comment by dougmet** [comment]
Yes that’s to do with how the tokens are split. There’s a setting in the filter to keep the parts. I can change that in the sole config
